public/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="./css/style.css">
+    <script src="./js/jquery-3.5.1.js"></script>
+    <title>Bowling</title>
+  </head>
+  <body>
+    <section>
+      <h1 id="score"></h1>
+      <fieldset>
+        <label for ="bowl_input">How many pins did you knock over?</label>
+        <input id="bowl_input" type="integer">
```

Comment:
Not sure if 'integer' is an input type. For me it's just displaying as a text box. Type 'number' would give you the up/down arrows